festflow/templates/festflow/base.html

```diff
@@ -22,11 +22,14 @@
 <body>
     {% navbar %}
     {% block body_contents %}{% endblock %}
+</body>
 
-    <!-- Jquery minified js -->
-    <script type="text/javascript" src='{% static "js/jquery.min.js"%}'></script>
+<!-- Jquery minified js -->
```

Comment:
why did you make this change? This is not correct. Put the scripts back to the end of the body and not outside it.